lib/http_clients/creditas/person_deletion_api.ex

```diff
@@ -13,6 +22,15 @@ defmodule HttpClients.Creditas.PersonDeletionApi do
     end
   end
 
+  @spec build_ack_payload(Acknowledgment.t()) :: map()
+  defp build_ack_payload(ack) do
```

Comment:
```suggestion
  defp build_ack_payload(ack) do
```
a gente geralmente não coloca spec em função privada, e :nail_care: é melhor jogar ela lá pra perto de onde ela é chamada :)